ios-cr-task/cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
```

Comment:
Missing module imports. To access UIKit features we need it. Add as below:

`import UIKit`